### PR TITLE
835395_Cursor_Position_UG

### PR DESCRIPTION
**Pull request by SivaranjithN**
Added Cursor position content for Autocomplete and ComboBox controls.

### CODE REVIEW

File: MAUI/Autocomplete/UI-Customization.md [hunk 1]
```diff
@@ -670,3 +670,27 @@ Completed event can be subscribed in C# also:
 The following image illustrates the result of the above code:
 
 ![.NET MAUI Autocomplete completed event](Images/UICustomization/CompletedEvent.png)
+
+## CursorPosition
+
+The cursor position in the input view can either be obtained or updated using the `CursorPosition` property in SfAutocomplete.
```
Comment (Chrispineagunja) [inline MAUI/Autocomplete/UI-Customization.md:676]: The cursor position in the input view can be obtained or updated using the `CursorPosition` property in the SfAutocomplete.

File: MAUI/ComboBox/UI-Customization.md [hunk 2]
```diff
@@ -698,3 +698,29 @@ Completed event can be subscribed in C# also:
 The following image illustrates the result of the above code:
 
 ![.NET MAUI ComboBox completed event](Images/UICustomization/CompletedEvent.png)
+
+## CursorPosition
+
+The cursor position in the input view can either be obtained or updated using the `CursorPosition` property in SfComboBox.
```
Comment (Chrispineagunja) [inline MAUI/ComboBox/UI-Customization.md:704]: The cursor position in the input view can be obtained or updated using the `CursorPosition` property in the SfComboBox.

File: MAUI/ComboBox/UI-Customization.md [hunk 3]
```diff
@@ -698,3 +698,29 @@ Completed event can be subscribed in C# also:
...
+using Syncfusion.Maui.Inputs;
+
+SfComboBox comboBox = new SfComboBox();
+comboBox.IsEditable = true;
+comboBox.CursorPosition = 4;
+
+{% endhighlight %}
+{% endtabs %}
+
+N> Cursor position support available for editable mode only and Two way binding not supported in Android platform.
```
Comment (Chrispineagunja) [inline MAUI/ComboBox/UI-Customization.md:726]: N> The cursor position support is available for editable mode only, and two-way binding is not supported in the Android platform.

**Review by Chrispineagunja — COMMENTED** [review]
Content review completed.

File: MAUI/Autocomplete/UI-Customization.md (hunk 1 above)
Comment (SivaranjithN) [reply MAUI/Autocomplete/UI-Customization.md]: Hi @Chrispineagunja,

Changed the content

File: MAUI/ComboBox/UI-Customization.md (hunk 3 above)
Comment (SivaranjithN) [reply MAUI/ComboBox/UI-Customization.md]: Hi @Chrispineagunja,

Changed the content.

**Review by PaulAndersonS — APPROVED** [review]

**Pull request merged**